src/module/cache-container/CacheContainer.ts

```diff
@@ -43,7 +43,7 @@ module.config(($stateProvider: ng.ui.IStateProvider) => {
     url: "/caches",
     templateUrl: "module/cache-container/caches/view/caches.html",
     controller: CachesCtrl,
-    controllerAs: "cachesCtrl",
+    controllerAs: "$ctrl",
```

Comment:
Was this changed due to preference, or is their a rationale?  Either way, if you're just going to use `$ctrl`, there is no need to define `controllerAs` as `$ctrl` is the default value.